test/e2e/e2e_suite_test.go

```diff
@@ -90,15 +94,21 @@ var _ = BeforeSuite(func() {
 	err = cs.List(context.TODO(), nodeList)
 	Expect(err).ToNot(HaveOccurred())
 
-	// get the EXTRA_NFTABLES_FILE if it not exist value is ""
-	val, exists := os.LookupEnv("EXTRA_NFTABLES_MASTER_FILE")
-	if exists {
-		extraNFTablesMasterFile = val
-	}
-
-	val, exists = os.LookupEnv("EXTRA_NFTABLES_WORKER_FILE")
+	portsToIgnoreCommatrix = nil
+	val, exists := os.LookupEnv("OPEN_PORTS_TO_IGNORE_FILE")
```

Comment:
Those are open ports we want to ignore in the comparison to to the ss. We don't add them to the commatrix

Reply:
In the nftable we are adding

Reply:
True, though we're adding them because we want to ignore them to avoid failure.
Maybe this name is no accurate but I think "extra ports" could be miss leading because we don't document them as EPS.